TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java

```diff
@@ -320,6 +324,16 @@ public void setPIDCoefficients(DcMotor.RunMode runMode, PIDCoefficients coeffici
         }
     }
 
+    public void setWeightedDrivePower(Pose2d drivePower) {
```

Comment:
This should include all of the logic from the localization test including the normalization.